bin/handler-opsgenie.rb

```diff
@@ -66,6 +72,11 @@ def process
     puts "opsgenie -- timed out while attempting to #{@event['action']} a incident -- #{event_id}"
   end
 
+  def description
+    return json_config['description'] unless json_config['description'].nil?
+    @event['check']['output']
```

Comment:
based on the fact that below it has a `.chomp` I would guess that we should do the same here unless you can confirm that there is no newline.

Reply:
The presence of a newline depends very much on the output of the check in question.  That said, I'll restore the `.chomp`; I don't think it will hurt anything.